astropy/stats/funcs.py

```diff
@@ -1178,6 +1178,14 @@ def _mpmath_kraft_burrows_nousek(N, B, CL):
     '''
     from mpmath import mpf, factorial, findroot, fsum, power, exp, quad
 
+    if type(N).__module__ == np.__name__:
+        N = N.item()
+
+    if type(B).__module__ == np.__name__:
+        B = B.item()
+
+    if type(CL).__module__ == np.__name__:
+        CL = CL.item()
```

Comment:
Would it not be easier to just do `N = mpf(float(N))`? That way it would work for anything that can be converted to a float. Checking the `__module__ of the type is a great idea I would have never thought of, but, unless really needed, is also pretty complicated.

Reply:
I guess all mpf does is convert the input into a float. So yeah that should work. I'm not sure why I didn't think of this XD